graph/src/components/ethereum/types.rs

```diff
@@ -416,25 +414,26 @@ impl Clone for EthereumCallData {
 /// Maximum block number supported: 2^63 - 1
 #[derive(Clone, Debug, PartialEq, Eq, Hash)]
 pub struct EthereumBlockPointer {
-    pub hash: H256,
+    pub hash: Bytes,
```

Comment:
Lets make this just a `Vec<u8>` so it is not tied to a type from the web3 crate.

Reply:
I actually made it `Box<[u8]>` to save a word of storage; I didn't use `scalar::Bytes`, since that has a scary `to_string()` that prefixes with `0x`, and using that seems too dangerous.

Reply:
Nice!

Reply:
It's funny how we use `Box<[u8]>` but nobody ever thinks of using `Box<str>`.